ethdev/flow/item_ipv4.go

```diff
@@ -55,6 +60,11 @@ func cvtIPv4Header(dst *C.struct_rte_ipv4_hdr, src *layers.IPv4) {
 	}
 }
 
+func setVersion(dst *C.struct_rte_ipv4_hdr, src *layers.IPv4) {
```

Comment:
can you come up this less ambiguous name of the function? `setVersion` sounds too general.